en/editor/publish/publish-bytedance-mini-game.md

```diff
@@ -0,0 +1,62 @@
+# Publish to ByteDance Mini Games
+
+ByteDance mini-game is developed based on the ByteDance full products, which does not require users to download, and is a brand-new game type that can be played on tap.
+
+The game submission, review and release process of the mini-games need to comply with the requirements and standard processes of the Byte official team. For specific information, please refer to the link at the end of the article.
+
+## Publish ByteDance with Cocos Creator 3D
+
+1. Download **ByteDance DevTools** on [ByteDance Official Document](https://microapp.bytedance.com/docs/zh-CN/mini-game/develop/developer-instrument/developer-instrument-update-and-download).
+
+2. Log in to [Developer Platform](https://microapp.bytedance.com/)to find appid. For details, please refer to the official details [Access Guide](https://microapp.bytedance.com/docs/zh-CN/mini-game/introduction/plugin-reference/set-up-mini-game)。
```

Comment:
2. Log in to [Developer Platform](https://microapp.bytedance.com/)to find appid. For details, please refer to the official details [Access Guide](https://microapp.bytedance.com/docs/zh-CN/mini-game/introduction/plugin-reference/set-up-mini-game)。
-->
2. Log in to the [Developer Platform](https://microapp.bytedance.com/) to find the `appid`. For details, please refer to the [Access Guide](https://microapp.bytedance.com/docs/zh-CN/mini-game/introduction/plugin-reference/set-up-mini-game) documentation.

Reply:
https://microapp.bytedance.com/docs/zh-CN/mini-game/introduction/plugin-reference/set-up-mini-game

English URL? If not, then please remove this doc from the Summary.md

Reply:
No English URL. But why should we remove it just because **ByteDance** don't have english docs? Developers can use tools to translate themselves.

Reply:
Then why did we write an English version?   It’s awkward. I don’t know enough about Bytedance to know if a non-chinese dev can even publish to it. 

Let’s ask for a decision.

Reply:
@pandamicro what do you think about the English ByteDance document not having English resources available in the resources section?

I wonder why we would write an English doc and then if the users need more help and they goto ByteDance can they even read the docs? Can users overseas even publish to BtyeDance or is it just or Chinese game companies? Thank you.

Reply:
the TOC will be modified and a dislaimer added to all of the platforms where Chinese docs are more prevalent than English docs.

Reply:
Per our conversation we need to add a disclaimer that English docs may not be available. I can do this once the TOC is organized and I will update all docs in the sub-TOC via a PR.

Reply:
@slackmoehrle  I have  change the summary about publish. All platforms that have only Chinese docs will be placed in the secondary menu so that they will be hide unless developers expands it themselves.